blocks/library/table/table-block.js

```diff
@@ -3,9 +3,28 @@ import BlockControls from '../../block-controls';
 import { Toolbar, DropdownMenu } from 'components';
 import { __ } from 'i18n';
 
+function isTableSelected( editor ) {
+	return editor.dom.getParent(
+		editor.selection.getStart( true ),
+		'table'
```

Comment:
Good point. DomUtils.getParent can take a root node as the third parameter so I'll set it to the editor container.